tensorflow/core/util/mkl_util.h

```diff
@@ -730,7 +730,7 @@ inline Status ConvertMklToTF(OpKernelContext* context,
   } catch (mkldnn::error& e) {
     string error_msg = "Status: " + std::to_string(e.status) +
                        ", message: " + string(e.message) + ", in file " +
-                       string(__FILE__) + ":" + std::to_string(__LINE__);
+                       string(__FILE__) + ":" + std::to_string(__LINE__);    
```

Comment:
Still got clang-format errors. This time because of the trailing spaces. (Sorry, I didn't notice it earlier.)
```suggestion
                       string(__FILE__) + ":" + std::to_string(__LINE__);
```